pkg/volume/util/util.go

```diff
@@ -581,10 +581,12 @@ func GetPluginMountDir(host volume.VolumeHost, name string) string {
 
 // IsLocalEphemeralVolume determines whether the argument is a local ephemeral
 // volume vs. some other type
+// Local means the volume is a local disk managed by kubelet
+// Ephemeral means the lifecycle of the volume is the same as the Pod
```

Comment:
nit: punctuation is missing after each of the three sentences.

I would say "Local means the volume is using storage from the local disk that is managed by kubelet.".

Reply:
Thanks. Fixed.